docs/maya/skin.md

```diff
@@ -145,18 +156,39 @@ In order to better visualise deformer constraints and attributes in the Maya vie
+ - **Slide Constraints**: For each vertex, a line will be drawn from the simulated mesh vertex to the corresponding point on the target mesh if its *Slide Constraints* weight is greater than 0.0.s
  - **Sliding Surface On Target**: For each vertex, lines will outline the target triangles within the reach of its *Max Sliding Distance*.
- - **Soft Constraints**: For each vertex, a line will be drawn from the simulated mesh vertex to the corresponding point on the reference mesh if its *Soft Constraints* weight is greater than 0.0.
+ - **Soft Constraints**: For each vertex, a line will be drawn from the simulated mesh vertex to the corresponding point on the target mesh if its *Soft Constraints* weight is greater than 0.0.
 
 <figure markdown>
   ![skin editor debug example](images/skin_debug.png)
-  <figcaption><b>Figure 5</b>: In gray the reference mesh, in orange the simulated skin. Debugger enabled displaying a test example with <i>Soft Constraints</i> coloured in green.</figcaption>
+  <figcaption><b>Figure 5</b>: In gray the target mesh, in orange the simulated skin. Debugger enabled displaying a test example with <i>Soft Constraints</i> coloured in green.</figcaption>
 </figure>
 
 <figure markdown>
   ![skin editor sliding surface debug](images/skin_debug_slide_surface.png)
-  <figcaption><b>Figure 6</b>: In gray the reference mesh, in red the simulated skin. Debugger enabled displaying the <i>Sliding Surface</i> coloured in green.</figcaption>
+  <figcaption><b>Figure 6</b>: In gray the target mesh, in red the simulated skin. Debugger enabled displaying the <i>Sliding Surface</i> coloured in green.</figcaption>
+</figure>
+
+## Advanced
+
+### Targets
+
+Once the AdnSkin deformer is created, it is possible to add and remove new targets to the system.
+
+- **Add targets**:
+    1. Select one or more mesh nodes to be assigned as targets to the AdnSkin.
+    2. Select the mesh that has the AdnSkin deformer applied.
+    3. Press the ![Add Targets](images/adn_add_skin_targets.png){style="width:4%"} button in the AdonisFX shelf or press *Add Targets* in the AdonisFX menu from the Edit Skin submenu.
+- **Remove targets**:
+    1. Select one or more mesh nodes that are assigned as targets to the AdnSkin.
+    2. Select the mesh that has the AdnSkin deformer applied.
+    3. Press the ![Remove Targets](images/adn_remove_skin_targets.png){style="width:4%"} button in the AdonisFX shelf or press *Remove Targets* in the AdonisFX menu from the Edit Skin submenu.
+    4. Alternatively, if only the mesh with the AdnSkin deformer is selected, when pressing the ![Remove Targets](images/adn_remove_skin_targets.png){style="width:4%"} button, all targets will be removed.
+
+<figure markdown>
+  ![skin adding multiple targets](images/skin_add_multiple_targets.png)
+  <figcaption><b>Figure 7</b>: Addition of multiple targets.</figcaption>
```

Comment:
I would suggest to clarify the use case that we are showing in the screenshot. 

```suggestion
  <figcaption><b>Figure 7</b>: Addition of multiple targets (Eg. Adding multiple muscle targets to a fascia/fat/skin simulation).</figcaption>
```

Reply:
Done.